Vates/VatesAPI/test/MDHWInMemoryLoadingPresenterTest.h

```diff
@@ -76,7 +89,8 @@ class MDHWInMemoryLoadingPresenterTest: public CxxTest::TestSuite
     EXPECT_CALL(*repository, canProvideWorkspace(_)).WillOnce(Return(false)); //No matter what the argument, always returns false.
 
     //Give a dummy name corresponding to the workspace.
-    MDHWInMemoryLoadingPresenter presenter(new MockMDLoadingView, repository, "_");
+    MDHWInMemoryLoadingPresenter presenter(
+        Mantid::Kernel::make_unique<MockMDLoadingView>(), repository, "_");
```

Comment:
`repository` in this and several other functions is a raw owning pointer (not visible in diff)!